cmake/modules/FindLibUSB.cmake

```diff
@@ -79,6 +79,7 @@ if(NOT LIBUSB_FOUND)
             file(DOWNLOAD
                 https://sourceforge.net/projects/libusb/files/libusb-1.0/libusb-${LIBUSB_WIN_VERSION}/libusb-${LIBUSB_WIN_VERSION}.7z/download
                 ${LIBUSB_WIN_ARCHIVE_PATH}
+                EXPECTED_MD5 cf3d38d2ff053ef343d10c0b8b0950c2
```

Comment:
This would change on every new release, wouldn't it?
Can we retrieve it from the repo? That would be very friendly for maintenance. ^^

Reply:
Libusb project produces new releases approx 1 per year. Also, we usually no need to update it quickly after release. So, I think, there is no problem for the maintainer to download archive and check MD5. And not, they don't provide checksums for archives.

Reply:
As mentioned before, it shall be ok, if there is one release per year on average.
Maybe it makes sense to have a reminder on that for yearly maintenance.